### DISCUSSION TITLE
Edit text in all Email Notifications

### DISCUSSION THREAD

**Discussion by MajorCreative** [mastodon/mastodon#18608] [Q&A]
Is it possible to update text in Email Notifications. 
My client wants to change the tone of voice in all emails.
For example we want to add text to Welcome email referencing our Instance  and our values etc.

**Comment by markcellus** [comment]
I was wondering the same thing. Should we just fork the project and edit the source files? Although maintaining a separate fork just for these smaller changes can be more trouble than it's worth.
